### PR TITLE
Ecosystem: updated process interval and batch sizes

### PR DESCRIPTION
**Pull request by nick-otter**

### CODE REVIEW

File: ecosystem.config.js [hunk 1]
```diff
@@ -87,8 +87,8 @@ module.exports = {
         WRITE_STORAGE_TYPE : "datatransfer.storage.FolderStorage",
         READ_STORAGE_TYPE : "datatransfer.storage.S3Storage",
         LOG_FILE_NAME : "s3-landing-to-local.log",
-        PROCESS_INTERVAL : 4,
-        MAX_FILES_BATCH : 25,
+        PROCESS_INTERVAL : 300,
```
Comment (david-wirelab) [inline ecosystem.config.js:90]: Can this not be a lower number so it is more frequent or is it controlled by a cronjob or pm?

File: ecosystem.config.js [hunk 2]
```diff
@@ -87,8 +87,8 @@ module.exports = {
...
+        MAX_FILES_BATCH : 200,
         FOLDER_DATE_OUTPUT : "False",
         LOG_LEVEL : "DEBUG",
```
Comment (david-wirelab) [inline ecosystem.config.js]: I'd change this to INFO

File: ecosystem.config.js [hunk 3]
```diff
@@ -105,7 +105,7 @@ module.exports = {
         WRITE_STORAGE_TYPE : "datatransfer.storage.FolderStorage",
         READ_STORAGE_TYPE : "datatransfer.storage.S3Storage",
         LOG_FILE_NAME : "s3-landing-to-local.log",
-        PROCESS_INTERVAL : 2,
+        PROCESS_INTERVAL : 60,
         MAX_FILES_BATCH : 1,
         FOLDER_DATE_OUTPUT : "False",
         LOG_LEVEL : "DEBUG",
```
Comment (david-wirelab) [inline ecosystem.config.js]: I'd change this to INFO

File: ecosystem.config.js [hunk 4]
```diff
@@ -105,7 +105,7 @@ module.exports = {
         WRITE_STORAGE_TYPE : "datatransfer.storage.FolderStorage",
         READ_STORAGE_TYPE : "datatransfer.storage.S3Storage",
         LOG_FILE_NAME : "s3-landing-to-local.log",
-        PROCESS_INTERVAL : 2,
+        PROCESS_INTERVAL : 60,
         MAX_FILES_BATCH : 1,
```
Comment (david-wirelab) [inline ecosystem.config.js:109]: Can this be upped?

Comment (nick-otter) [reply ecosystem.config.js:109]: Reflects Prod.

File: ecosystem.config.js (hunk 1 above)
Comment (nick-otter) [reply ecosystem.config.js:90]: Reflects Prod.

**Review by david-wirelab — APPROVED** [review]

**Pull request merged**